recipes/mysql-connector-c/all/conanfile.py

```diff
@@ -81,6 +84,13 @@ def package(self):
             os.remove(f)
         tools.rmdir(os.path.join(self.package_folder, "docs"))
 
+    @property
+    def _require_stdcpp_library(self):
+        return self.settings.compiler in ("gcc",) and not self.options.shared
```

Comment:
out of curiosity, why does gcc require libstdc++, but clang does not require std lib at all ?

Reply:
I did not test this recipe using clang but it will probably also require a similar system library.

I will probably have to do @uilianries 's suggestion (https://github.com/conan-io/conan-center-index/pull/1445#discussion_r414823980) by only removing `self.settings.cpp_info.cppstd` and adding `c++` or `stdc++` if `self.settings.compiler.libcxx` is in ["libstdc++", "libstdc++11", "libc++"]